python-libraries/nanover-omni/src/nanover/omni/ase_omm.py

```diff
@@ -93,6 +110,11 @@ def load(self):
         )
 
     def reset(self, app_server: NanoverImdApplication):
+        """
+        Reset the simulation to its initial conditions, reset IMD interactions, and reset frames to begin with topology
```

Comment:
```suggestion
        Reset the simulation to its initial conditions, reset IMD interactions, and reset frames to initial positions
```

Reply:
Just a suggestion!

Reply:
Have clarified this last part--it refers to the fact that the sequence of topology->normal->normal->normal frames will begin again